tests/isVisible.spec.ts

```diff
@@ -0,0 +1,109 @@
+import { mount } from '../src'
+
+describe('isVisible', () => {
+  const Comp = {
+    template: `<div><span v-show="show" /></div>`,
+    props: {
+      show: {
+        type: Boolean
+      }
+    }
+  }
+
+  it('returns false when element hidden via v-show', () => {
```

Comment:
*suggestion*: WDYT of adding one more pair of tests `returns false when ancestor is hidden via v-show` to capture our intent - we're detecting visibility not only on entire element itself

Reply:
Sure, good idea.

Reply:
done!